ext/oj/dump.c

```diff
@@ -1003,9 +1003,9 @@ void oj_dump_false(VALUE obj, int depth, Out out, bool as_ok) {
 void oj_dump_fixnum(VALUE obj, int depth, Out out, bool as_ok) {
     char      buf[32];
     char *    b              = buf + sizeof(buf) - 1;
-    long long num            = rb_num2ll(obj);
+    long long num            = NUM2LL(obj);
     int       neg            = 0;
-    int       cnt            = 0;
+    long      cnt            = 0;
```

Comment:
This will remove warning message at compilation with clang.

```
../../../../ext/oj/dump.c:1039:35: warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]
    cnt = sizeof(buf) - (b - buf) - 1;
        ~ ~~~~~~~~~~~~~~~~~~~~~~~~^~~
```